netbox_onboarding/migrations/0003_onboardingtask_change_logging_model.py

```diff
@@ -0,0 +1,19 @@
+from django.db import migrations, models
```

Comment:
Can we squash the 0002 and 0003 migrations together before merging to develop?

Reply:
@carbonarok This change should not be a part of this PR